data/tree/values tester/f1b5321b-d504-4ae3-9eb9-8dae3426f137.json

```diff
@@ -1,22 +1,22 @@
 {
   "name": "data/tree/Values tester",
-  "description": "Tests whether all of the given data structure values are the same as in the reference",
+  "description": "Tests for deep equality for `data` and `reference`.\n\nExample:\n1. {\"foo\":{\"bar\":\"baz\"}}@0 is received via `data`\n2. {\"foo\":{\"bar\":\"baz\"}}@0 is received via `reference`\n3. true@0 is sent via `all values are same`",
   "interface": {
     "inputs": {
       "e468b571-c70a-453c-a6d9-6532e7ac947a": {
         "description": "The dictionary to test",
-        "type": "dictionary",
+        "type": "{string:any}",
```

Comment:
Rather ``` any[] or {string: any} ```
Also the description should rather use the term "tree" instead of "dictionary"

Reply:
Also rename the port to "tree" to be consistent with port naming of other nodes in this namespace?